apps/arweave/src/ar_blacklist_middleware.erl

```diff
@@ -74,10 +75,10 @@ reset_rate_limit(IpAddr) ->
 	ets:delete(?MODULE, {rate_limit, IpAddr}),
 	ok.
 
-increment_ip_addr(IpAddr) ->
+increment_ip_addr(IpAddr, Increment) ->
```

Comment:
Maybe have a `increment_ip_addr(IpAddr) -> increment_ip_addr(IpAddr, 1);` as well so we don't update the code where the arg is `1`?